test/infrastructure/docker/internal/loadbalancer/config.go

```diff
@@ -23,19 +23,25 @@ import (
 	"sigs.k8s.io/kind/pkg/errors"
 )
 
-// ConfigData is supplied to the loadbalancer config template
+// ConfigData is supplied to the loadbalancer config template.
 type ConfigData struct {
 	ControlPlanePort int
 	BackendServers   map[string]string
 	IPv6             bool
 }
 
-// DefaultConfigTemplate is the loadbalancer config template
-const DefaultConfigTemplate = `# generated by kind
+// ConfigTemplate is the loadbalancer config template.
+const ConfigTemplate = `# generated by kind
 global
   log /dev/log local0
   log /dev/log local1 notice
   daemon
+  # limit memory usage to approximately 18 MB
```

Comment:
I would appreciate a comment that explains where this limit came from, because I think it's far from self-explanatory :laughing: 

```suggestion
  # limit memory usage to approximately 18 MB
  # (see https://github.com/kubernetes-sigs/kind/pull/3115)
```

Reply:
Will add - thanks for getting that merged btw! Very useful fix :smile: